app/airq/commands/menu.py

```diff
@@ -11,6 +11,7 @@ def handle(self) -> typing.List[str]:
             "1. Details and recommendations",
             "2. Current AQI",
             "3. Hazebot info",
+            "4. Give us feedback",
```

Comment:
I think we need to wait to resolve the issue of where to put this before we merge it down.